.github/workflows/ros2foxy.yml

```diff
@@ -86,4 +86,54 @@ jobs:
         USER_EMAIL: 'ros-contributions@amazon.com'
         USER_NAME: 'ros-contributions'
       if: ${{ github.event_name == 'schedule' && contains(steps.robot_ws_build.outcome, 'success') && contains(steps.simulation_ws_build.outcome, 'success') }} # Update app-manifest version number if the build was successful
-  
\ No newline at end of file
+  
+
+
+  log_workflow_status_to_cloudwatch:
+  runs-on: ubuntu-latest
+  container:
+    image: ubuntu:bionic
+  needs:
+  - build_and_bundle_ros2
```

Comment:
the job is named build_and_bundle_ros2_foxy in this workflow

Reply:
good catch